src/Protocol/Protocol_1_8.cpp

```diff
@@ -1420,18 +1420,26 @@ void cProtocol_1_8_0::SendSpawnMob(const cMonster & a_Mob)
 void cProtocol_1_8_0::SendStatistics(const cStatManager & a_Manager)
 {
 	ASSERT(m_State == 3);  // In game mode?
-
+	std::vector<Statistic> send = {};
+	for (auto e = a_Manager.GetDirtyStatistics()->begin(); e != a_Manager.GetDirtyStatistics()->end(); e++)
```

Comment:
Prefer ranged for loops, for (const auto Item : DirtyStats())

Reply:
this improves readability quite a bit. I Knew that you could do this in java but this also works for c++ - yeah 👍 

done